docs/changelog.rst

```diff
@@ -222,6 +222,10 @@ HTTP frontend
   :confval:`http/static_dir`, which has been deprecated since 1.0. (Fixes:
   :issue:`1463`, PR: :issue:`1706`)
 
+- Add option ``default_app`` to redirect ``http:hostname``:``http:port`` to a specific app.
```

Comment:
Use ``` :confval:`http/static_dir` ```, and it'll link to the docs. Then you can remove the second sentence from the changelog.

Also, maybe change the it to "...to redirect to a specific app instead of Mopidy's web app list."?